protocol/avdl/stellar1/local.avdl

```diff
@@ -50,6 +50,12 @@ protocol local {
   // supported currencies, returned from getDisplayCurrenciesLocal.
   void changeDisplayCurrencyLocal(int sessionID, AccountID accountID, OutsideCurrencyCode currency);
 
+  record UserSettings {
+    boolean acceptedDisclaimer; // whether the user has accepted the usage disclaimer
+  }
+  UserSettings getUserSettingsLocal(int sessionID);
+  void setAcceptedDisclaimerLocal(int sessionID); 
```

Comment:
I don't see these RPCs in the spec. Is the `get` one going to be superseded by `GetWalletSettings` ?

Reply:
The names are confusing to me right now, if `Wallet` will eventually refer to the feature as a whole (and`Account` to what is now called a `Wallet`), then yes it'll become `getWalletSettings`.